src/index.html

```diff
@@ -4,180 +4,280 @@
+          гостиниц в седоне</p>
+        <a class="booking-link__bottom blue-bottom">Найти гостиницу</a>
+      </section>
+
+      <section class="map">
+        <h2 class="visually-hidden">Карта</h2>
+        <iframe class="map__google" width="320" height="350" title="Sedona Location"
+          src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d52385.18173350387!2d-111.83633831988399!3d34.85437145319606!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x872da132f942b00d%3A0x5548c523fa6c8efd!2z0KHQtdC00L7QvdCwLCDQkNGA0LjQt9C-0L3QsCA4NjMzNiwg0KHQqNCQ!5e0!3m2!1sru!2sru!4v1708657346327!5m2!1sru!2sru"
+          width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy"
+          referrerpolicy="no-referrer-when-downgrade">
+        </iframe>
+      </section>
+
+    </main>
+
+    <footer class="footer">
+      <section class="our-tag">
+        <h2 class="contact__tag title"> #VISITSEDONA </h2>
+      </section>
+      <section class="social">
+        <h2 class="visually-hidden">Социальные сети</h2>
+        <ul class="social__list">
+          <li class="social__item">
+            <a class="social__link" href="#">
+              <svg class="social__svg" role="img" aria-label="Icon «Twitter»" viewBox="0 0 18 15" width="18" height="15">
+                <use xlink:href="#twitter-svg"></use>
+              </svg>
+            </a>
+          </li>
+          <li class="social__item">
+            <a class="social__link" href="#">
+              <svg class="social__svg" role="img" aria-label="Icon «Facebook»" viewBox="0 0 9 19" width="9" height="19">
+                <use xlink:href="#facebook-svg"></use>
+              </svg>
+            </a>
+          </li>
+          <li class="social__item">
+            <a class="social__link" href="#">
+              <svg class="social__svg" role="img" aria-label="Icon «Youtube»" viewBox="0 0 20 14" width="20" height="14">
+                <use xlink:href="#youtube-svg"></use>
+              </svg>
+            </a>
+          </li>
+        </ul>
+      </section>
+      <section class="copyright">
+        <h2 class="visually-hidden">created by</h2>
+        <p class="copyright__text subtitle">Разработано</p>
+        <a class="copyright__link" href="https://htmlacademy.ru/intensive/htmlcss">
+          <svg class="html-academy-svg" role="img" aria-label="Logo «HTML Academy»" viewBox="0 0 27 34" width="27" height="34">
+            <use xlink:href="#html-academy-svg"></use>
+          </svg>
+        </a>
+      </section>
+
+
+    </footer>
 
+  </div>
+  <script>
+    var navMain = document.querySelector('.nav');
+var navToggle = document.querySelector('.nav__toggle');
 
-  </footer>
+navToggle.addEventListener('click', function() {
+if (navMain.classList.contains('nav--open')) {
+navMain.classList.remove('nav--open');
+} else {
+navMain.classList.add('nav--open');
+}
+});
+  </script>
```

Comment:
You always can refer to the [MDN](https://developer.mozilla.org/ru/docs/Web/HTML/Element/script) If you have questions about `JavaScript`, `HTML` or `CSS`.